modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,110 @@
+
+If the TinyMCE document is large enough to produce a scroll bar, clicking the caret prefixing a footnote entry scrolls the document back to the line containing said entry’s superscripted footnote number.
+
+NOTE: Although the viewport scrolls back to the line containing the footnote number, the insertion point is not changed. Users must move the insertion point by, for example, clicking the I-beam pointer somewhere in the now-scrolled document.
+
+
+== Footnotes and {productname} Previews
+
+*File > Preview* presents footnote-specific in-document links, but these links do not work in the Preview.
+
+
+== Footnotes and exporting {productname} documents to PDF
+
+In-document footnote links are not preserved when a {productname} document is exported to a PDF file.
+
+The footnotes and footnote text are preserved, but the clickable links between them are not. In an exported PDF file, the superscripted footnote numbers are not clickable.
+
+And the carets set between a footnote number and footnote text, which act as links back to a superscripted footnote number in a {productname} document, are removed in the exported PDF file.
+
+
+== Formatting superscripted footnote numbers
+
+. Superscripted footnote numbers cannot be inline-formatted
++
+If a selected string includes a superscripted footnote number and inline-formatting is applied to that string, the superscripted footnote number does not take on the applied formatting.
++
+Inline-formatting includes setting a string to *Bold* or _Italic_ or `monospaced`.
+
+. Superscripted footnote numbers can be block-formatted
++
+If a selected string includes a superscripted footnote number and block-formatting is applied to that string, the superscripted footnote number takes on the applied formatting.
++
+Block-formatting includes making a selection a Heading of any level or setting the selection to `Preformatted`.
+
+== Formatting footnote text
+
+. Footnote text can be inline-formatted
++
+Footnote text can be formatted using inline-formatting options such as *Bold*, _Italic_, or `monospaced`.
+
+. Footnote text cannot be block-formatted
++
+Footnote text cannot, for example, be made into a Heading, or set as `Preformatted`.
+
+== Known issue
+
+Footnote entries are not automatically removed after removing an image caption containing a footnote.
+
+When an image caption with an attached footnote is removed (by unchecking the _Show caption_ checkbox in the **Insert/Edit Image* dialog), the associated footnote text should be automatically removed.
+
+As of this release, however, this automatic removal does not happen.
+
+The orphan footnote text can be deleted manually. It can also be removed by clicking the `footnotesupdate` button in the toolbar.
```

Comment:
I feel like this is more general than specifically image captions. It would be better to specify this as a use case for the `updatefootnotes` button. Seeing as this "bug" occurs from any kind of content removal (of a footnote) directly through the DOM, which is the purpose of the button. Another example of the footnotes section not automatically being updated is when you insert a footnote in a table, and then delete the row/column it exists within; the footnotes section will not automatically update in this case either. Yet another use for the updatefootnotes button.

Reply:
To quote my commit message:

> Turned known issue re footnotes and image captions into new section on the update footnotes button.